tls/s2n_alerts.c

```diff
@@ -63,6 +63,18 @@ static bool s2n_alerts_supported(struct s2n_connection *conn)
     return conn && !conn->quic_enabled;
 }
 
+static bool s2n_is_warning(struct s2n_connection *conn, uint8_t level, uint8_t type)
+{
+    /* Only TLS1.2 considers the warning level. The warning level field is
+     * considered deprecated in TLS1.3. */
```

Comment:
Nit: Can we update this comment to include the exception:

The warning level field is  considered deprecated in TLS1.3 (`except in user_cancelled alerts`)?

Reply:
No, TLS1.3 never considers the alert level. That field is deprecated.

Instead, TLS1.3 chooses to treat user_canceled as a warning based on the fact that it's user_canceled. We only consider the type, not the level.